libs/utils/android/system.py

```diff
@@ -359,7 +359,7 @@ def gfxinfo_get(target, apk_name, out_file):
                     GET_FRAMESTATS_CMD.format(apk_name, out_file))
 
     @staticmethod
-    def monkey(target, apk_name, event_count=1):
+    def monkey(target, apk_name, event_count=1, pct_syskeys=0):
```

Comment:
That would make sense, especially if we consider the idea to add more and more functionalities.
In that case however I would suggest you to have a look at [devlib](https://github.com/ARM-software/devlib). 

In the future, we are planning to move most of the LISA's Android API within the devlib library, where it probably makes more sense for them to be. Indeed, devlib is a target abstraction library and thus it would probably be the best place to host a complete new monkey wrapper class.

For me it would be ok to start by merging this simple fix/extension you already got in LISA.
In a second stage we can try to add a more comprehensive class to devlib, where it could be probably hosted in:
https://github.com/ARM-software/devlib/blob/master/devlib/module/android.py